Dockerfile

```diff
@@ -1,9 +1,15 @@
 FROM golang:1.12.5-alpine as builder
 ARG VERSION
+ENV GO111MODULE=on
 RUN mkdir -p /go/src/github.com/TimothyYe/bing-wallpaper
 WORKDIR /go/src/github.com/TimothyYe/bing-wallpaper
 RUN cd /go/src/github.com/TimothyYe/bing-wallpaper
 COPY . .
+RUN apk --no-cache add git \
```

Comment:
I think just enable `GO111MODULE=on` is OK enough, then go module will download all these dependencies automatically.

Reply:
Just tried it out, the installation of git is required here.